contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,121 @@
+
+<ProductScreenshot
+  imageLight={PricingComparison} 
+  imageDark={PricingComparisonDark} 
+  alt="New PostHog pricing compared to Mixpanel" 
+  classes="rounded"
+/>
+
+## What's the downside of sending this type of event?
+
+There are some limitations with the new event type. You cannot:
+
+- Filter on persons.
+- Create cohorts filtered on person properties.
+- Target by person properties for feature flags, A/B tests and surveys.
+- Receive initial UTM values for tracking marketing campaign performance (only the most recent).
+- View a person’s profile in the app, or query the persons table in our SQL insights.
+- Use [groups functionality](/docs/product-analytics/group-analytics) on standard events (without person profiles).
+
+If you need the above functionality, just continue to send person information with your events (no need to change anything), for events where the specific functionality is needed. 
+
+Person profiles are not an all-or-nothing thing; you can use them on any traffic where you need more detailed information, and skip them when you don't need that detail.
+
+## How do I start using this new processing option?
+
+For most people, it only requires a simple config change to include `process_persons: "identified_only"`. 
+
+This will track anonymous traffic without person profiles, and start collecting person information any time an identifying action is taken (i.e. using `identify()`, `group()`, setting person properties with `$set`, etc).
+
+You can get instructions for this and other use cases in our [persons documentation](/docs/data/persons).
+
+## FAQs
+
+### Is this change sustainable?
+
+Yes! 
+
+Here's why:
+
+* Out of all the major competitors we have, we believe we are by far the most efficient – we are 100% inbound, so we don't need to pay for an outbound sales team, and grow through word of mouth.
+
+* This pricing still reflects us making a positive margin on every event sold.
+
+* We suspect this will increase our word-of-mouth growth and increase our retention, so we are likely to make more revenue with this change, but at a lower margin.
+
+* The majority of our customers use multiple products that we offer, so we wind up making a little money off a lot of products - our competitors are not as wide, so we believe long term we can undercut.
+
+### Why aren't you making the events with person properties cheaper too?
+
+* Events with person properties cost a lot more for us to ingest and query. Allowing customers to choose to not send person properties for certain event types means we can cut our own costs of providing our product to our customers. 
+
+* This aligns our interests with our customers, and means we can make a positive margin on all events ingested (beyond our free tier), so it is sustainable but not greedy!
```

Comment:
```suggestion
* Because events with person profiles cost a lot more for us to ingest and query. Allowing customers to choose to not send person profiles for certain event types means we can cut our own costs of providing our product to our customers. 

* This aligns our interests with our customers, and means we can make a positive margin on all events ingested (beyond our free tier) so it is sustainable but not greedy!

* We do hope to lower the person profiles cost at lower volumes in the somewhat near future.
```

Restores @raquelmsmith's suggestion